docs/index.md

```diff
@@ -0,0 +1,5 @@
+# Auto-GPT
```

Comment:
not sure if this generates a side bar with a Table of Contents but if not, make one here

Reply:
The mkdocs site does yes.
![image](https://user-images.githubusercontent.com/1500012/233773226-54a224d0-e9cb-43e9-a6bb-c7ebecd7fe1a.png)